cluster/images/kubemark/Makefile

```diff
@@ -15,8 +15,21 @@
 # build Kubemark image from currently built binaries containing both 'real' master and Hollow Node.
 # This makefile assumes that the kubemark binary is present in this directory.
 
+REGISTRY?="gcr.io"
+PROJECT?="google_containers"
+
 all:
-	docker build --pull -t gcr.io/$(PROJECT)/kubemark .
-	gcloud docker -- push gcr.io/$(PROJECT)/kubemark
+	gcloudpush
+
+build:
+	docker build --pull -t $(REGISTRY)/$(PROJECT)/kubemark .
+
+gcloudpush:
+	build
```

Comment:
i'm no makefile expert, but if you want push to always do a build step, i believe that you'd put build on the line above after the :